tools/osbuilder/rootfs-builder/ubuntu/config.sh

```diff
@@ -5,7 +5,7 @@
 OS_NAME=ubuntu
 # This should be Ubuntu's code name, e.g. "focal" (Focal Fossa) for 20.04
 OS_VERSION=${OS_VERSION:-focal}
-PACKAGES="chrony iptables"
+PACKAGES="chrony iptables cryptsetup-bin e2fsprogs"
```

Comment:
@arronwy @fidencio just thinking aloud...when we attempt to merge CCv0 -> main it will be a nightmare as we will need to search for all CC specific snippets of code. For example here, I don't believe we will include disk crypto utils for the non-CC image, won't we?

So I was wondering if we could start "marking" those places, or use `if` guards. While ago, when @stevenhorsman , @fidencio and I started the work to merge the branches, I introduced the `KATA_BUILD_CC` variable (see https://github.com/kata-containers/tests/pull/4675) for the purpose of `if KATA_BUILD_CC=yes then do_some_CC_specific_stuff` logic. Would make sense to start using that variable right way?

wdyt?

Reply:
Yes, if we finally need merge CCv0 branch with main branch, we can do it as early as possible